sync-core/src/main/java/com/cloudant/sync/datastore/SavedHttpAttachment.java

```diff
@@ -36,7 +36,6 @@
     private URI attachmentURI;
     private int size;
     private byte[] data;
-    private Encoding encoding;
```

Comment:
encoding is stored in the Attachment super class, is encoded length worth storing? I don't think its useful since there isn't an API to ask for it